canonical-web-and-design/ubuntu.com
# Refrseh core page (due June 16th)

## Done

- Page refresh on `/core`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/core or https://ubuntu-com-11703.demos.haus/core
    - Be sure to test on mobile, tablet and desktop screen sizes
- Compare page against [copy doc](https://docs.google.com/document/d/1UFpefDRSAcndNBnWp0hDyzqk74aciqhBWJEi05htWmU/edit#)

## Issue / Card

Fixes [#5428](https://github.com/canonical-web-and-design/web-squad/issues/5428)
